### ISSUE TITLE
Display Author in Inbox view

### ISSUE THREAD

**Issue by NoahTheDuke** [GitHawkApp/GitHawk#1885]
Currently, the issue/pr title, number, and number of comments is shown on the main Inbox view (and in View All). It would be nice to see the author as well, listed in the white space underneath the title. This would mirror the web interface. If you’re feeling frisky, you could also layer the participating profile pics in the same way as the web interface.

**Comment by rnystrom** [comment]
@NoahTheDuke author that caused the notification, or author of the issue/PR? (Assuming the latter)

Dig the idea, but we have to be careful with space here. Not sure where to put it.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by NoahTheDuke** [comment]
Author (OP) of the issue/PR.

Currently, in the app Project Issues/PRs view, it mirrors the web interface which is includes labels and is honestly very clunky looking. Leaving out the labels, you could fit the OP underneath the title.

I mocked up an example (below) from a screenshot of "View All" (as I don't have any notifications, lol). Included are three pairs of ideas: The first of each pair shows the time since the issue/PR was opened (which mirrors the app Project Issues/PRs view) and the second does not (just the username of the OP). The first pair is the basic idea, the second and third pairs include the "users who've interacted with this" avatars as listed in the web interface's Notifications page in two different locations.

![example](https://user-images.githubusercontent.com/603677/41860626-17e2d876-786d-11e8-80bd-2c52b912cc68.png)

**Comment by NoahTheDuke** [comment]
No worries! Thanks so much for all the hard work you put in!

**Comment by BasThomas** [comment]
Didn’t want to close this; it’s just a feeling! 😃

Thanks for bringing up the issue 😊

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by NoahTheDuke** [comment]
My apologies for the quick close, and you're welcome! I wish I had a Mac, cuz I'd love to contribute; I use this app every single day.

**Comment by dimirc** [comment]
> author that caused the notification, or author of the issue/PR? (Assuming the latter)

@rnystrom I think will be more natural is the author that caused the notification, actually the Github web version works like that and personally this helps me to know what notifications to check first based on who produced them. 

On my opinion the author of the Issue/PR shouln't be that important at the Inbox View, as we're checking notifications and not issues/PR, so we want to know who created the former

**Comment by dimirc** [comment]
Pls correct me if I’m wrong but I was checking the source code to try to do a PR for this change and looks like the app is using the v3 REST api to get notifications instead of the v4 graphQL api?

**Comment by BasThomas** [comment]
Yeah, that could be. I think there are some discrepancies between the v3 and v4 API in places; this might be the reason. Not entirely sure though.
